package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "webtask-runtime",
-  "version": "1.6.0",
+  "version": "1.6.1",
```

Comment:
Let's go with a minor since it's introducing a new 'feature'.

Reply:
Done